# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/modules/FaviconLoader.jsm
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c browser/modules/FaviconLoader.jsm
--- a/browser/modules/FaviconLoader.jsm	Tue Feb 12 17:58:19 2019 +0100
+++ b/browser/modules/FaviconLoader.jsm	Tue Feb 12 16:08:25 2019 +0000
@@ -144,7 +144,7 @@
     this.stream = new BufferedOutputStream(this.dataBuffer.getOutputStream(0), STREAM_SEGMENT_SIZE * 2);
 
     try {
-      this.channel.asyncOpen2(this);
+      this.channel.asyncOpen(this);
     } catch (e) {
       this._deferred.reject(e);
     }